HelpSource/Tutorials/A-Practical-Guide/PG_Cookbook05_Using_Samples.schelp

```diff
@@ -35,21 +76,22 @@ SynthDef(\bell, { |out, accent = 0, amp = 0.1, decayScale = 1|
 			{ ExpRand(0.1, 0.4) } ! 4
 		], exc, freqscale: accent + 1, decayscale: decayScale);
 	DetectSilence.ar(sig, doneAction: Done.freeSelf);
-	Out.ar(out, sig ! 2)
+	Out.ar(out, sig.dup)
 }).add;
 )
 
 (
-TempoClock.default.tempo = 0.35953685899971 * 4;
+TempoClock.tempo = 1.4381474359988;
```

Comment:
how about setting `start` and `end` variables and writing `TempoClock.tempo = ((end - start) / numBeats).reciprocal`? that seems like a better recommended practice than a hardcoded number.

Reply:
OK, with a tighter math expression.